private/bufpkg/buffeature/buffeature.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package buffeature provides known feature flags for the Buf CLI.
+package buffeature
+
+import "github.com/bufbuild/buf/private/pkg/app"
+
+// Available Buf feature flags.
+var (
+	// TamperProofing enables the Buf tamper proofing feature.
+	// This enables sending of a manifest and blobs when pushing modules.
+	// Additionally, it enables support for consuming a manifest and blobs when downloading modules.
+	TamperProofing = app.FeatureFlag{Name: "BUF_FEATURE_TAMPER_PROOFING", Default: false}
```

Comment:
Doesn't feel necessary - this feels like it should be something along the lines of `tamperProofingEnvKey = "BUF_ENABLE_TAMPER_PROOFING"` that is just within the `bufcli` package